src/DotNetWorker.Core/Definition/FunctionParameter.cs

```diff
@@ -22,6 +22,16 @@ public FunctionParameter(string name, Type type)
         {
         }
 
+        /// <summary>
+        /// Creates an instance of the <see cref="FunctionParameter"/> class.
+        /// </summary>
+        /// <param name="name">The parameter name.</param>
+        /// <param name="type">The <see cref="System.Type"/> of the parameter.</param>
+        /// <param name="defaultValue">Default value of the parameter.</param>
+        public FunctionParameter(string name, Type type, object? defaultValue): this(name, type, defaultValue, ImmutableDictionary<string, object>.Empty)
```

Comment:
nit: follow the existing formatting. Change to:
``` csharp
public FunctionParameter(string name, Type type, object? defaultValue) 
    : this(name, type, defaultValue, ImmutableDictionary<string, object>.Empty)
```

Reply:
Fixed formatting as suggested.